README.md

```diff
@@ -167,6 +167,30 @@ export default function App() {
 }
 ```
 
+### Get the access token
+
+Sometimes it is useful to obtain the access token directly (e.g. to make API requests to another service).
```

Comment:
This is a little vague, could use a better example on what you'd do with the access token.